.github/workflows/python-tests.yml

```
@@ -11,18 +11,18 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.8", "3.11"]
```

We still want to test each version between:

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11"]
```